coalib/bears/requirements/GoRequirement.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+
+
+class GoRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    requirements from ``go``, without using the manager name.
+    """
+
+    def __init__(self, package, version="", flag=""):
+        """
+        Constructs a new ``GoRequirement``, using the ``PackageRequirement``
+        constructor.
+
+        >>> pr = GoRequirement('setuptools', '19.2', '-u')
+        >>> pr.manager
+        'go'
+        >>> pr.package
+        'setuptools'
+        >>> pr.version
+        '19.2'
+        >>> pr.flag
+        '-u'
+
+        :param package: A string with the name of the package to be installed.
+        :param version: A version string. Leave empty to specify latest version.
+        :param flag:    A string that specifies any additional flags, that
+                        would be used with ``flag``.
```

Comment:
i have absolutely no idea, but thats how we install it for .ci, so it must be there, i didnt want to find out what it does because thats not the point of this pr :P